doc/claw.rst

```diff
@@ -0,0 +1,212 @@
+.. Copyright (C) 2017-2018, Science and Technology Facilities Council, UK
+
+.. _claw:
+
+CLAW
+====
+
+CLAW is a Fortran source-to-source compiler written by V. Clement of
+CSCS. It is normally driven by the use of CLAW pragmas added to source
+code. PSyclone makes use of CLAW for the purpose of transforming
+kernel code to make it suitable/performant for e.g. use with OpenACC.
+For this purpose, PSyclone includes code that interfaces with CLAW and
+enables it to be used from within transformation scripts.
+
+Prerequisites
+-------------
+
+In order to use the CLAW functionality you will obviously need the CLAW
+compiler installed. This in turn requires the Omni compiler.
+You will also need Jython (a Java implementation of Python).
+
+Installation/Configuration
+--------------------------
+
+Omni Compiler
+^^^^^^^^^^^^^
+
+The Omni Fortran compiler is available from http://omni-compiler.org/.
+PSyclone has been tested with version 1.2.2 but should work with any
+version that is more recent than this. In order to build it you will
+first need to install yacc, flex and the Java development kit.
+
+Once Omni has been built and installed it will need to be available
+on your PATH so that PSyclone can find it.
+
+Jython
+^^^^^^
+
+Jython is a Java implementation of Python and is available from
+http://www.jython.org/downloads.html. Once you have downloaded the
+installer jar, installation is performed by doing:
+
+::
+
+    java -jar jython-installer-2.7.0.jar
+
+You will then need to edit the PSyclone configuration file and specify
+the location of the jython.jar file.
+
+CLAW
+^^^^
+
+The version of the CLAW compiler used by PSyclone is available from
+https://github.com/stfc/claw-compiler.  See the INSTALL.md file for
+installation instructions.  Once it is installed, you will need to
+manually copy the Python interface module into the `lib` directory of
+the Claw installation:
+
+::
+
+    cp claw-compiler/omni-cx2x/src/claw/python/ClawTransform.py <claw-install-root>/lib
```

Comment:
I could not find a file called ClawTransform.py or even a directory called omni-cx2x.

Reply:
My apologies - this goes back to the fact that Valentin has re-structured CLAW and I got carried-away and pulled his changes into the stfc version of CLAW. The file you need is now:

    cx2t/src/claw/python/ClawTransform.py

(On the `psyclone` branch of stfc/claw-compiler)